array-methods.js

```diff
@@ -88,7 +139,13 @@ var higherStateSums = null;
   if true set `areStatesInHigherStateSum` to `true`
   otherwise set it to `false`
  */
-var areStatesInHigherStateSum = null;
+const areStatesInHigherStateSum = Object.entries(stateSums)
+  .filter(function (arr) {
```

Comment:
Just a note, the filter here does not need to be on the next line but I have seen in both ways in the wild.